accepted/0000-workspaces-command.md

```diff
@@ -0,0 +1,31 @@
+- Start Date: 2017-05-04
+- RFC PR:
+
+# Summary
+
+This document specify a new command, that can be used to execute subcommands inside a project workspaces.
+
+# Motivation
+
+With the addition of the Workspace feature, it will become handy to be able to execute commands inside other workspaces than the one in the current directory.
+
+# Detailed design
+
+This RFC suggests to add the two following commands:
+
+```
+$> yarn workspace <project-name> pwd
+$> yarn workspace <project-name> <command-name> ...
```

Comment:
We've been using `lerna run` and `lerna exec` for a little while. I haven't heard of anyone confused by it. I think we can mitigate that with documentation if it does become an issue.

npm/Yarn `run` commands are already open to cross-platform incompatibilities since you could put `pwd` in `package.json#scripts` and have the same cross-platform problems. People (including me) still think its useful regardless.